core/arch/riscv/riscv.mk

```diff
@@ -44,6 +44,7 @@ endif
 
 CFG_RISCV_SBI	 ?= n
 CFG_RISCV_M_MODE ?= y
+CFG_RISCV_S_MODE ?= n
```

Comment:
Not needed, the checks after sets it according the `CFG_RISCV_M_MODE `.

Reply:
OK, I'll drop it.